doc_src/auto_examples/plot_fit_fake_data.py

```diff
@@ -0,0 +1,37 @@
+"""
+Example for fitdata function
+============================
+This example creates fake data with noise
+then fits the exponential with the fitdata
+function.
+"""   
+from pyspecdata import *
+from pyspecdata import fitdata 
+from sympy import symbols
+import numpy as np
+# %%
+# creating a fake data recovery curve
+tau = nddata(r_[0:2:100j], 'tau')
```

Comment:
@guinneak198 : did you create this file, or was it copied by sphinx from a different location?

Reply:
I made it

Reply:
it doesn't seem right that you have to create the file in the examples directory, and then manually copy it here -- it seems like you are missing some kind of automated tool you should be using.